common/addons/structure-terrascript-loader/src/main/java/com/dfsek/terra/addons/terrascript/script/functions/BlockFunction.java

```diff
@@ -35,17 +34,19 @@ public class BlockFunction implements Function<Void> {
     protected final Platform platform;
     private final Map<String, BlockState> data = new HashMap<>();
     private final Returnable<Boolean> overwrite;
+    private final Returnable<Boolean> physics;
     private final Position position;
     
     public BlockFunction(Returnable<Number> x, Returnable<Number> y, Returnable<Number> z, Returnable<String> blockData,
-                         Returnable<Boolean> overwrite, Platform platform, Position position) {
+                         Returnable<Boolean> overwrite, Returnable<Boolean> physics, Platform platform, Position position) {
```

Comment:
no it's not breaking because nothing outside the add-on should ever use this. Changing the signatures of the internals of an add-on is not breaking

Reply:
if "nothing outside the addon should ever use this", then it should be package-private, annotated with smth like @Internal, or in a   `*.impl.*` package, otherwise it is a public api.

Reply:
I still disagree this should be considered breaking but I've pushed a stub anyways to resolve this

Reply:
In cade I'm going yo revert the stub commit to clean up the code

Reply:
In cade I'm going to revert the stub commit to clean up the code